pkg/logql/log/parser_test.go

```diff
@@ -978,8 +978,8 @@ func Test_unpackParser_Parse(t *testing.T) {
 			l, _ := j.Process(0, tt.line, b)
 			sort.Sort(tt.wantLbs)
 			require.Equal(t, tt.wantLbs, b.LabelsResult().Labels())
-			require.Equal(t, tt.wantLine, l)
 			require.Equal(t, string(tt.wantLine), string(l))
+			require.Equal(t, tt.wantLine, l)
```

Comment:
This change means that you get human-readable strings when this fails rather than a pile of bytes.